docs/global.html

```
@@ -0,0 +1,2446 @@
+
+
+        </dl>
+
+
+
+
+
+
+
+
+        <h4 class="name" id="p2pAddNodes"><span class="type-signature"></span>p2pAddNodes<span class="type-signature"> :Array.&lt;string></span></h4>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)